src/test/scala/computerdatabase/ComputerWorld.scala

```diff
@@ -0,0 +1,42 @@
+import scala.concurrent.duration._
+
+import java.util.concurrent.ThreadLocalRandom
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class ComputerWorld extends Simulation {
+
+  val httpProtocol = http
+    .baseURL("http://computer-database.gatling.io")
+    .acceptHeader("""text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8""")
+    .acceptEncodingHeader("""gzip, deflate""")
+    .acceptLanguageHeader("""en-gb,en;q=0.5""")
+    .contentTypeHeader("application/x-www-form-urlencoded")
```

Comment:
remove this, it doesn't have to be sent for the get requests, and it's automatically generated for the post one